doc/source/getting_started/install.rst

```diff
@@ -95,7 +95,7 @@ as well::
     - `fiona`_ provides binary wheels with the dependencies included for Mac and Linux,
       but not for Windows. Alternatively, you can install `pyogrio`_ which does
       have wheels for Windows.
-    - `pyproj`_, `rtree`_, and `shapely`_ provide binary wheels with dependencies included
+    - `pyproj`_, and `shapely`_ provide binary wheels with dependencies included
```

Comment:
```suggestion
    - `pyproj`_ and `shapely`_ provide binary wheels with dependencies included
```

Reply:
This is actually also not up to date anymore, fiona now has windows wheels as well

Reply:
-> https://github.com/geopandas/geopandas/pull/3036 (can merge that afterwards to deal with the conflict there)